Automattic/jetpack
# Jetpack Forms: Add/week month props on sent message tracks event

A very minor change adding week and month of the year properties to the tracks event when a response email is sent to the owner of a Jetpack Forms form. The date is taken from the publish date of the post containing the form.

There was an "is_block" check there which depended on an old, deprecated attribute called hasFormSettingsSet that was removed in #15362 but the is_block check wasn't removed. AFAICT no events were sent from forms created since then.

Testing instructions:
Apply patch
Add a Jetpack form to a test page, fill it in and submit it.
Look at the live tracks event page for the Jetpack contact_form_block_message_sent and watch out for your message. It may take up to an hour to appear. The properties should include the new post_w and post_m properties.

Discussion: pcsBup-S8-p2

## Does this pull request change what data or activity we track or use?
Yes, we are tracking when a message is emailed to the owner of an embedded Jetpack Form. This patch adds the week/month numbers of the publish date of the parent post to that tracking. Separately, we already have the publish date.
